tests/orion/api/test_block_schemas.py

```diff
@@ -216,6 +227,22 @@ async def test_delete_block_schema_for_system_block_type_fails(
             == "Block schemas for protected block types cannot be deleted."
         )
 
+    async def test_delete_block_schema_for_system_block_type_does_not_fail_for_older_clients(
+        self, session, system_block_type, client_with_unprotected_block_api
+    ):
+        block_schema = await models.block_schemas.create_block_schema(
+            session=session,
+            block_schema=schemas.actions.BlockSchemaCreate(
+                fields={}, block_type_id=system_block_type.id
+            ),
+        )
+        await session.commit()
+
+        response = await client_with_unprotected_block_api.delete(
+            f"/block_schemas/{block_schema.id}"
+        )
+        assert response.status_code != 403
```

Comment:
Should we assert that status code is 204 like the other tests that have a successful deletion?